src/modules/atommodel/particles/RadixUniqueParticle.ts

```diff
@@ -1,7 +1,7 @@
 import { RadixSerializer, RadixParticle, includeDSON, includeJSON, RadixAddress, RadixResourceIdentifier } from '..';
 
 @RadixSerializer.registerClass('UNIQUEIDPARTICLE')
-export class RadixTokenClassParticle extends RadixParticle {
+export class RadixUniqueParticle extends RadixParticle {
```

Comment:
👍 This name is much, much better!

Reply:
I know, right? I was hit by a wave of inspiration and there it was